gdcdictionary/schemas/treatment.yaml

```diff
@@ -24,7 +24,7 @@ systemProperties:
 links:
   - name: diagnoses
```

Comment:
Are there existing data using this? If so we will need to migrate the data?

Reply:
yes, existing data are there